db/seeds.rb

```diff
@@ -80,6 +77,26 @@ def random_record_for_org(org, klass)
   end
 end
 
+# ----------------------------------------------------------------------------
+# Request Units
+# ----------------------------------------------------------------------------
+
+%w(pack pad flat).each do |name|
+  RequestUnit.create!(organization: pdx_org, name: name)
+end
+
+pdx_org.items.each_with_index do |item, i|
+  if item.name == 'Pads'
+    %w(pad pack).each { |name| item.request_units.create!(name: name) }
```

Comment:
Along those lines, I guess this should only add `pack` and not `pad`

Reply:
Updated.